extensions/users-permissions/models/user.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+const axios = require('axios');
+
+const subscribeToMailingList = async (values) => {
+  const { data } = await axios.post(`https://barmga-lambda.nyaladev.com/email-subscribe`, values);
```

Comment:
let's make the URL `https://barmga-lambda.nyaladev.com` in an environment variable 

```
const url = process.env.EMAIL_SUBSCRIBE_URL || `https://barmga-lambda.nyaladev.com/email-subscribe`:
const { data } = await axios.post(url, values);
```